.env.example

```
@@ -130,8 +130,11 @@ WATSONX_ENDPOINT=
 WATSONX_PROJECT_ID=
 
 # LLM Provider configuration. Providers can be "anthropic", "watsonx", "ibm" or "ollama".
-LLM_PROVIDER=
-LLM_MODEL=
+# Chat model efectivo: onboarding UI → ./config (y flows Langflow). No usa GRANITE_MODEL.
+# Dev default (CPU/RAM limitada). Con GPU: granite4.1:8b o granite4.1:30b en onboarding/settings.
+# Opcional como semilla antes del primer onboarding (se sobrescribe al completarlo):
+LLM_PROVIDER=ollama
+LLM_MODEL=granite4.1:3b
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LLM_MODEL key should go before the LLM_PROVIDER key

(UnorderedKey)